fatips/103.md

```diff
@@ -0,0 +1,152 @@
+| FATIP | Title                      | Status   | Category | Author                                     | Created  |
+| ----- | -------------------------- | -------- | -------- | ------------------------------------------ | -------- |
+| 103   | FAT Entry Signing Standard |   .      | Core     | Adam S Levy \<adam@canonical-ledgers.com\> | 1-4-2019 |
+
+
+
+# Summary
+
+This standard defines how FAT entries should be signed with a given set of
+ed25519 keys so as to completely avoid replay attacks both within and across
+chains.
+
+Specifically this standard currently mitigates these known replay attack
+vectors:
+- Replay across chains.
+- Delayed transmission replays. i.e. A signed transaction is held back and
+  submitted at a later date.
+- External ID reordering.
+- Additional External IDs.
+
+# Motivation
+
+As all FAT token standards rely on the uniqueness of the entry hash as a
+validity rule, replay attacks can occur when the entry hash can be altered
+without invalidating the entry itself. All FAT token standards can adopt this
+standard for their signing requirements to avoid replay attacks.
+
+
+# Specification
+
+## External ID Structure
+
+The External IDs of an entry contain a timestamp salt and the RCD/Signature
+pairs required to validate the entry.
+
+| Index | Encoding | Example | Details  |
+| - | - | - | - |
+| 0       | Decimal String | "1546443301" or `0x31 35 34 36 34 34 33 33 30 31` | A Unix second timestamp that MUST be within +/-12 hours of the entry's on-chain timestamp. By adjusting the timestamp into the future or past, a user can control how long before a signed transaction will expire. |
+| 1       | Raw Bytes | `0x01 49 bb 7b 5f bc fc fc 04 75 32 2b a1 4c 80 77 7d 29 bd 37 8c 33 8a 7f f7 fe d0 88 38 20 b1 92 10` | RCD 0 |
+| 2       | Raw Bytes | `0x5f 06 e4 87 16 98 44 1e d9 3e da 2a e8 25 d2 a7 d1 59 f4 04 c8 d8 1b 9b 88 ab 11 79 fd c8 f8 de e4 db a3 89 75 d5 43 a2 8b 20 13 5d 24 23 f6 1c 70 72 b5 f5 2f 02 98 64 b2 bc 9e 6d ba 7a ee 02` | Signature 0 |
+| ...     | ...  | ...  | ... |
+| `X*2+1` | Raw Bytes | ... | RCD `X` |
```

Comment:
nitpicking: I'd recommend using `N` instead of `X`, as integers are generally denoted by n in mathematics field. Actually at the very end of the document you used `N`.